Article/Article.less

```diff
@@ -39,10 +39,31 @@
     transform: translate(-50%);
     background-color: white;
   }
+  .closeButton{
+    position: absolute;
+    bottom: 0;
+    right: 25%;
+    font-size: 12px;
+    color: grey;
+    cursor: pointer;
+    transform: translate(-50%);
+    background-color: white;
+
+  }
 }
 
 .article-open {
-  height: 400px;
+  height: auto;
+  
+}
+.article-close{
+  display:none;
+}
```

Comment:
Im not sure if this is the intended function of your buttons, but if you click expand followed by expand again, it closes. If you click expand followed by read, the article disappears.